subgraph/queries/tokens.py

```diff
@@ -101,86 +95,86 @@ def fetch_token_balances(
-                "lastID": last_id,
-                "blockNumber": {"number": block_number},
-            }
-            next_page = client.execute(query, variable_values=variables)
-            if len(next_page["tokenBalances"]) == 0:
-                continue_fetching = False
-            else:
-                last_id = next_page["tokenBalances"][-1]["id"]
-                console.log(
-                    f"Fetching {len(next_page['tokenBalances'])} token balances"
-                )
-                for entry in next_page["tokenBalances"]:
-                    address = entry["id"].split("-")[0]
-                    amount = int(entry["balance"])
-                    if amount > 0:
-                        if entry["token"]["symbol"] == "BADGER":
-                            badger_balances[address] = amount / DECIMAL_MAPPING[chain]
-                        if entry["token"]["symbol"] == "DIGG":
-                            fragment_balance = digg_utils.shares_to_fragments(
-                                int(amount)
-                            )
-                            digg_balances[address] = float(fragment_balance) / 1e9
-    except Exception as e:
-        send_error_to_discord(
-            e, "Error in Fetching Token Balance", "Subgraph Error", chain
-        )
-        raise e
-
+    while continue_fetching:
+        variables = {
+            "firstAmount": increment,
+            "lastID": last_id,
+            "blockNumber": {"number": block_number},
+        }
+        next_page = client.execute(query, variable_values=variables)
+        if len(next_page["tokenBalances"]) == 0:
+            continue_fetching = False
+        else:
+            last_id = next_page["tokenBalances"][-1]["id"]
+            console.log(
+                f"Fetching {len(next_page['tokenBalances'])} token balances"
+            )
+            for entry in next_page["tokenBalances"]:
+                address = entry["id"].split("-")[0]
+                amount = int(entry["balance"])
+                if amount > 0:
+                    if entry["token"]["symbol"] == "BADGER":
+                        badger_balances[address] = amount / DECIMAL_MAPPING[chain]
+                    if entry["token"]["symbol"] == "DIGG":
+                        fragment_balance = digg_utils.shares_to_fragments(
+                            int(amount)
+                        )
+                        digg_balances[address] = float(fragment_balance) / 1e9
     return badger_balances, digg_balances
 
 
-# flake8: noqa: E501
-def fetch_fuse_pool_balances(client: Client, chain: Network, block: int) -> Dict[str, Dict]:
-    if chain != Network.Ethereum:
-        console.log("Fuse pools are only active on ETH")
-        return {}
-
-    client = SubgraphClient("fuse", chain)
-    ctoken_data = {
-        "fBADGER-22": {
-            "underlying_contract": "0x3472A5A71965499acd81997a54BBA8D852C6E53d",
+def fetch_fuse_token_info() -> Dict:
```

Comment:
If this is a completely stale info, why not move it to constants?

Reply:
Or just not to constant, but move this dict to the top of the file, don't need a function to return stale info